### PR TITLE
Extend account creation methods

### PR DESCRIPTION
**Pull request by ekoval**
#55

### CODE REVIEW

File: sitewit/services.py [hunk 1]
```diff
@@ -104,6 +106,9 @@ def create_account(self, url, user_name, user_email,
         if site_id:
             data['clientId'] = site_id
 
+        if mobile_phone:
+            data['mobilePhone'] = mobile_phone
```
Comment (RayeN) [inline sitewit/services.py]: We could probably use `_remove_nones` here to simplify the code.

Comment (RayeN) [reply sitewit/services.py]: And I think we should support mobile_phone in the model as well.

**Comment by RayeN** [comment]
changelog entry?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**